streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -198,6 +220,11 @@ private RocksDB openDB(File dir, Options options, int ttl) throws IOException {
         }
     }
 
+    //Visible for testing
+    boolean isPrepareForBulkload() {
```

Comment:
I do not think we need this variable at all, since as mentioned above when initiating it will always be true, and that is the only place this variable is ever read.